faq.md

```diff
@@ -243,9 +246,32 @@ Most of its capabilities are on par with or inferior to our custom FPGA.
 There are issues when putting it on the system bus related to CPU read attempt speed call and answer
 The Propeller costs the same as if not more than the faster FPGA
 
+## Seriously, if the X16 doesn't have X, I don't want it.
+
+At this stage the design has been ratified with hardware in people's hands and
+changes to the base hardware are very unlikely.
+
+Engineering is ultimately an exercise in compromise. To meet the goals of the
+project at a cost target, not everything and the kitchen sink could be included.
+Likewise, the X16 represents a stable platform with which to write amazing
+software. If it had too many features (like a bunch of FPGA accelerators) it
+would be harder to use and compatibility, testability, etc. would have been more
+difficult.
+
+Building a community around the X16 is just important as the hardware and given
+the educational goals of the X16, this meant the base system is static and
+simple.
+
+That said, one of the benefits of the X16 is that it does have a fully capable
+expansion bus available for folks to augment and extend it in many ways. This
+allows for a lot of possibilities! Even CPU accelerator cards are possible!
+
+If the X16 isn't your cup of tea, that's totally ok! There are plenty of retro
+options, both new and old, available and we hope you find one that suits you!
+
```

Comment:
I'm going to approve the change overall, but I'm still not comfortable with this portion, which is largely opinion. I'd rather take this out and talk about whether we can write something a little more objective that still addresses the influx of "why doesn't it have ...?" questions and comments.